tools/src/wyvern/target/corewyvernIL/support/ModuleResolver.java

```diff
@@ -318,35 +338,25 @@ private Module loadContinuation(File file, String qualifiedName, TypedAST ast, b
                         oldDefDecl.getFormalArgs(),
                         oldDefDecl.getType(),
                         oldDefDecl.getBody(),
-                        oldDefDecl.getLocation()
+                        oldDefDecl.getLocation(),
+                        oldDefDecl.getEffectSet()
                 );
 
                 // Wrap in an object
                 program = new New(defDecl);
                 List<FormalArg> formalArgs = ((DefDeclType) defDecl.getDeclType()).getFormalArgs();
 
+
                 // Perform quantification lifting if possible
                 final GenContext newGenCtx = extendGenContext(genCtx, dependencies);
                 final New liftResult = QuantificationLifter.liftIfPossible(newGenCtx, program, isLifted);
                 if (liftResult != null) {
                     program = liftResult;
                 }
 
-//                for(FormalArg arg : formalArgs) {
-//                    Set<EffectDeclType> effects = getEffects(arg, newGenCtx);
-//                    for(DeclType declType : arg.getType().getStructuralType(newGenCtx).getDeclTypes()) {
-//                        if(declType instanceof EffectDeclType) {
-//                            EffectDeclType effectDecl = (EffectDeclType) declType;
-//                            effectDecl.adapt(View.from(program, newGenCtx));
-//                            System.out.println(effectDecl);
-//                        }
-//                    }
-//                }
-
-//                List<DeclType> firstDeclTypes = (h.getFormalArgs().get(0).getType().getStructuralType(newGenCtx).getDeclTypes());
-//                List<DeclType> secondDeclTypes = (h.getFormalArgs().get(1).getType().getStructuralType(newGenCtx).getDeclTypes());
-//                System.out.println(firstDeclTypes);
-//                System.out.println(secondDeclTypes);
+                // Perform effect annotation check
+//                boolean result = EffectAnnotationChecker.isAnnotated(genCtx, program);
```

Comment:
Is this check necessary? If so, why is it commented out?

Reply:
I have already checked if the module is effect annotated on line 310. Therefore, I should remove this line, instead of commenting it out.